libs/matdbg/src/CommonWriter.h

```diff
@@ -214,13 +226,15 @@ const char* toString(backend::Precision precision) noexcept {
 }
 
 inline
-const char* toString(backend::SamplerFormat format) noexcept {
-    switch (format) {
-        case backend::SamplerFormat::INT: return "int";
-        case backend::SamplerFormat::UINT: return "uint";
-        case backend::SamplerFormat::FLOAT: return "float";
-        case backend::SamplerFormat::SHADOW: return "shadow";
+const char* toString(backend::StencilOperation stencilOp) noexcept {
```

Comment:
Couldn't these functions be marked as constexpr? Currently we don't call them in constant evaluated contexts, but could come handy later.

Reply:
They could be but I'd rather keep the convention of the surrounding Filament code. Having this new one as an exception would look weird and changing all Filament calls would make Filament version bumps potentially more cumbersome.